identity/webapp/pages/api/registration-form.ts

```diff
@@ -0,0 +1,29 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { generateNewToken, decodeToken } from '../../src/utility/jwt-codec';
+
+export default (req: NextApiRequest, res: NextApiResponse): void => {
+  if (req.method === 'POST') {
+    const {
+      state,
+      redirectUri,
+      firstName,
+      lastName,
+      termsAndConditions,
+      sessionToken,
+    } = req.body;
```

Comment:
We should be checking that these are populated as expected and returning a 400 if not